src/parsers/manifestjson.js

```diff
@@ -261,15 +261,15 @@ export default class ManifestJSONParser extends JSONParser {
     // super helpful error. We'll tidy it up a bit:
     if (error && error.message) {
       const lowerCaseMessage = error.message.toLowerCase();
-      if (lowerCaseMessage === 'should match some schema in anyof') {
+      if (lowerCaseMessage === 'must match a schema in anyof') {
         // eslint-disable-next-line no-param-reassign
         error.message = 'is not a valid key or has invalid extra properties';
       }
     }
 
     const overrides = {
-      message: `"${error.dataPath}" ${error.message}`,
-      dataPath: error.dataPath,
+      message: `"${error.instancePath || '/'}" ${error.message}`,
```

Comment:
I couldn''t find any explicit callout for this change in their v6-to-v8 migration doc (or v8 release notes) but it seems that for the error related to the nested property the error used to be associated to the nested property while now it is instead associated to the parent level.

And so when the error is associated to the root level, the instance path seems to undefined instead of '/'.